### PR TITLE
AnalyticsReporter: Require opt-in for conversion tracking

### PR DESCRIPTION
**Pull request by data-enabler**

### CODE REVIEW

File: src/core/analytics/analyticsreporter.js [hunk 1]
```diff
@@ -44,10 +51,14 @@ export default class AnalyticsReporter {
...
     return new HttpRequester().beacon(
-      `${ANALYTICS_BASE_URL}/realtimeanalytics/data/answers/${this._businessId}`,
+      `${this._baseUrl}/realtimeanalytics/data/answers/${this._businessId}`,
       {
         'data': event.toApiEvent()
       }
     );
   }
+
+  setConversionTrackingEnabled (enabled) {
```
Comment (b1lly) [inline src/core/analytics/analyticsreporter.js]: nit: isEnabled

**Review by b1lly — DISMISSED** [review]
lgtm 💃

File: src/core/analytics/analyticsreporter.js (hunk 1 above)
Comment (data-enabler) [reply src/core/analytics/analyticsreporter.js]: Done

**Comment by data-enabler** [comment]
Rebased onto v0.9.3

**Review by b1lly — APPROVED** [review]

**Pull request merged**